### PR TITLE
readmes: first pass

### PR DESCRIPTION
**Pull request by divyenduz**
Related https://github.com/prisma/prisma2-e2e-tests/issues/52

Fixes https://github.com/prisma/prisma2-e2e-tests/issues/46

### CODE REVIEW

File: platforms/netlify-ci-build/README.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # Introduction
 
-Prisma and Netlify integration
+Prisma and Netlify (beta build) integration.
```
Comment (janpio) [inline platforms/netlify-ci-build/README.md]: Can we link somewhere so one can know what "beta build" refers to?

File: platforms/netlify-ci-build/README.md [hunk 2]
```diff
@@ -1,7 +1,8 @@
...
 
 # Friction Points
 
 - https://github.com/netlify/build/issues/730
```
Comment (janpio) [inline platforms/netlify-ci-build/README.md]: Can we put a super short description of the Friction point here? In this case probably "`binaryTargets` needs to be `[...]`" or similar.

File: platforms/netlify-zisi/README.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# Introduction
+
+Netlify's current build bot uses an older version of zip-it-and-ship-it. This is however, overrideable using an env var called `ZISI_VERSION`.
```
Comment (janpio) [inline platforms/netlify-zisi/README.md]: What is "build bot"?
Link `zip-it-and-ship-it` to https://github.com/netlify/zip-it-and-ship-it

File: platforms/netlify-zisi/README.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+
+This build tries to use that env var and deploy Prisma client with minimal configuration with Netlify.
+
+# Friction points
+
+- The Netlify CI build bot did not pick up the correct `ZISI_VERSION`.
+- This test is merged (to avoid a long-running branch) but the workflow doesn't run yet
```
Comment (janpio) [inline platforms/netlify-zisi/README.md]: It doesn't? Where is it disabled?

File: platforms/netlify-zisi/README.md (hunk 3 above)
Comment (divyenduz) [reply platforms/netlify-zisi/README.md]: They call their CI build `build bot`.

File: platforms/netlify/README.md [hunk 5]
```diff
@@ -1,48 +1,8 @@
...
-
-Since for Netlify, we are bundling manually, we can remove the local binary before creating the zip bundle for deployment. 
-
-## Friction Overview
-
-- Prisma schema config
-  - `platform` needs to be explicitly set
-  - `output` folder diverges from `@prisma/client`
-- Cumbersome build steps required
+- Since Netlify's current build bot doesn't use the newer version of `zip-it-and-ship-it`, we need to bundle the binaries ourselves. See `build.sh`.
```
Comment (janpio) [inline platforms/netlify/README.md]: see above

File: platforms/netlify-ci-build/README.md (hunk 1 above)
Comment (divyenduz) [reply platforms/netlify-ci-build/README.md]: Yes, leaving a note for myself here: https://build-beta.netlify.com/

File: platforms/netlify-zisi/README.md (hunk 4 above)
Comment (divyenduz) [reply platforms/netlify-zisi/README.md]: ah, good catch. I just renamed it and didn't disable it, will do that 👍

File: platforms/netlify-zisi/README.md (hunk 3 above)
Comment (divyenduz) [reply platforms/netlify-zisi/README.md]: Changed it to build CI.

**Pull request merged**